auth-web/src/locales/en.json

```diff
@@ -76,7 +76,7 @@
   "passwordResetFailureText": "An error occurred while attempting to reset this password.\nPlease try again later.",
   "tos_updated": "We have updated our Terms of Use. Please review and accept the new terms to proceed.",
   "createBCRegistriesAccount": "Create a BC Registries Account",
```

Comment:
This should probably read:

"Create a BC Registries and Online Services Account"